### ISSUE TITLE
notifications are not showing up on IOS

### ISSUE THREAD

**Issue by easazade** [fluttercommunity/flutter_downloader#351]
notifications work fine on android but they are not showing on IOS. I've done all the configuration required in xcode adding background capabilities `Background fetch` & `Remote Notification` also added `Background Process`. but the notifications are still not showing. 

FYI : I have only tested the app on IOS Simulator so far.

**Comment by zahidshaikh08** [comment]
Its not supported by plugin yet they clearly mentioned in documentation this thread should be feature enhancement because notification in iOS will be great although android is working very great.

**Comment by easazade** [comment]
@zahidshaikh08 This feature will be an enhancement to this plugin in a most definite way 🦉

**Comment by wurstnudl** [comment]
> Its not supported by plugin yet they clearly mentioned in documentation this thread should be feature enhancement because notification in iOS will be great although android is working very great.

No, it's NOT clearly mentioned in the documentation. It says on the first page: "Localize notification messages: the plugin will send a notification message to notify user in case all files are downloaded"

It might be a good idea to fix this.

**Comment by easazade** [comment]
@wurstnudl thanks for mentioning that. the documentation is a little old. for example the IOS configuration for this plugin doesn't work all the time. the correct config can be found in some one of the issues

**Comment by awaisahmad94** [comment]
Guys any fix for this??
